docs/en/enterprise-edition/content-collections/cloud-and-software-inventory/iac-resources.adoc

```diff
@@ -51,4 +58,36 @@ For the IaC Resources Inventory, filters provide a granular way to customize you
 * *Framework:* Allows you to focus on IaC assets based on the specific set of tools, libraries, and conventions used in their implementation. It provides insights into the underlying technology stack and methodology employed.
 * *IaC Resource Tag:* Offers insights into the tags associated with the IaC resource.
 * *Code Provider:* Indicates the cloud service provider associated with the asset.
-* *Last Modified By:* Use this filter to track and view changes made to the IaC asset based on the user who most recently modified it. It provides transparency in the development and maintenance process.
\ No newline at end of file
+* *Last Modified By:* Use this filter to track and view changes made to the IaC asset based on the user who most recently modified it. It provides transparency in the development and maintenance process.
+* *Resource Classes*: Filter IaC misconfigurations based on IaC resource classes. For more information, see <<resource-class,Resource Classes>> below.
+
+[#resource-class]
+=== Resource Classes
+
+IaC resource classes provide a structured approach to categorizing infrastructure resources based on their type, function, or other relevant criteria. This categorization enables efficient filtering and management of assets within the IaC inventory.
+
+NOTE: Currently, only Terraform and CloudFormation frameworks support IaC resource classes.
+
+To filter assets by IaC resource class:
+
+. Under *Inventory*, select select the *IaC Resources* tab.  
+. Select a framework from the list in the inventory table.
+
+
+. Click *Add Filter* > *Resource Class*.
++
+image::cloud-and-software-inventory/inventory-resource-class1.1.png[]
+
+. Click on the *Resource Class* filter > select an option (see below) from the menu.
```

Comment:
I would suggest a minor rewrite here the UI path style is limited clickable elements in the UI. So here I would suggest:

Select an option from the *Resource Class* filter dropdown as shown below.